### PR TITLE
Updated README to specify that this is a fork

### PR DESCRIPTION
**Pull request by tzumainn**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,26 +2,43 @@
...
+
+.. code-block:: bash
+
+  vbmc add <node name>
+  vbmc start <node name>
+
+Run `vbmc --help` to view additional options.
 
 
 Supported IPMI commands
```
Comment (samuel-zuk) [inline README.rst:44]: I think in the spirit of things, you might want to add a disclaimer that this is in development, experimental, or something to that effect and/or specify which IPMI functions are supported by the tool.

**Review by samuel-zuk — CHANGES_REQUESTED** [review]
Looks good! It might be a good idea to revise the 'supported IPMI commands' section to reflect the changes we make/intend to make to the code, but other than that this is great.

File: README.rst (hunk 1 above)
Comment (tzumainn) [reply README.rst:44]: Fair enough! I've added "development" in the summary at the top. As far as I'm aware, all the IPMI commands here should work.

**Comment by tzumainn** [comment]
Change made - let me know what you think!

File: README.rst (hunk 1 above)
Comment (samuel-zuk) [reply README.rst:44]: Ah, okay. Sounds good!

**Review by samuel-zuk — APPROVED** [review]
Looks good to me!

**Pull request merged**